Cargo.toml

```diff
@@ -215,7 +215,7 @@ ropey = "1.6.1"
 rust-lapper = "1.1.0"
 ryu-js = "1.0.2"
 saphyr = "0.0.4"
-schemars = "0.8.22"
+schemars = { git = "https://github.com/camc314/schemars", rev = "394e41b086ed5191bdc6063092a4a8aea9a70666" }
```

Comment:
This invalidates cache every time you build (correct me if I'm wrong), so publish it to crates.io. Rename to `oxc-schemars`?

Reply:
> Note: [crates.io](https://crates.io/) does not allow packages to be published with dependencies on code published outside of [crates.io](https://crates.io/) itself ([dev-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) are ignored). See the [Multiple locations](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) section for a fallback alternative for git and path dependencies

good point, will fix